packaging/osx/buildpackage.sh

```diff
@@ -12,9 +14,12 @@ if [ -e "OpenRA.app" ]; then
     exit 2
 fi
 
+wget https://github.com/OpenRA/OpenRALauncherOSX/releases/download/${LAUNCHER_TAG}/launcher.zip || exit 3
```

Comment:
It doesn't really matter, because I gave up trying to keep the gnuisms out of these build scripts years ago, and they don't work under OSX.